src/string.ts

```diff
@@ -49,6 +49,18 @@ export const dash = (str: string): string => {
   })
 }
 
+/**
+ * Joins all string arguments in a Pasacal case fashion
```

Comment:
'Pascal' here is misspelled as 'Pasacal'.

Reply:
Thanks!